coding/python_style_guide.rst

```diff
@@ -783,3 +783,36 @@ Iterators and generators SHOULD be used to iterate over large data sets efficien
 -----------------------------------------------------------------------------------
 
 Use iterators, generators (classes that act like iterators) and generator expressions (expressions that act like iterators) to iterate over large data sets efficiently.
+
+`if False:` and `if True:` SHOULD NOT be used
```

Comment:
Like the other sections, could you add a reference anchor before this section. Probably

```
.. _style-guide-py-feature-flags:
```